### PR TITLE
Add getTierPricesForChannelCode

### PR DESCRIPTION
**Pull request by pierre-H**
As for Sylius price

### CODE REVIEW

**Comment by mamazu** [comment]
Looks good. Just needs the interface to also specify the method as well.

**Comment by pierre-H** [comment]
Interface updated

File: src/Traits/TierPriceableTrait.php [hunk 1]
```diff
@@ -62,6 +62,22 @@ public function getTierPricesForChannel(ChannelInterface $channel): array
         });
     }
 
+    /**
+     * Returns the tier prices only for one channel
+     *
+     * @param $code
```
Comment (mamazu) [inline src/Traits/TierPriceableTrait.php]: Missing the type hint.

**Comment by mamazu** [comment]
@pierre-H Thanks for your work.

**Pull request merged**